test/fixtures/locales-dotted/en-us/translations.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  "node1": {
```

Comment:
Your fixture files for "dotted" aren't dotted, that's why the directories aren't equal.